src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -129,7 +129,7 @@ def acr_connected_registry_create(cmd,  # pylint: disable=too-many-locals, too-m
     )
 
     try:
-        return client.create(subscription_id=subscription_id,
+        return client.begin_create(subscription_id=subscription_id,
```

Comment:
why do you need to change the code from xyz to begin_xyz? What is the difference between the two?

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/track_2_migration_guidance.md#long-running-operation-function-name-change, no behavior change, only name change.

Reply:
Refer to this [Guide](https://devblogs.microsoft.com/azure-sdk/migrating-python-management-libraries/#long-running-operations-lro-method-names)